server/resources.py

```diff
@@ -4,15 +4,15 @@
 from models.prospects import ProspectModel, prospect_schema, prospects_schema
 from flask_restful import Resource
 from flask import request, session
-import io, csv
+import io, csv, redis, json
 
 
 def login_required(f):
     @wraps(f)
     def wrap(*args, **kwargs):
         auth_header = request.headers.get('Authorization')
         if auth_header:
-            auth_token = auth_header.split(" ")[1]
+            auth_token = auth_header.split()[1]
```

Comment:
Pretty sure we still need the split done on the space?? when we get the token it looks like this below: 
<img width="591" alt="Screen Shot 2020-07-31 at 11 02 32 AM" src="https://user-images.githubusercontent.com/22139916/89063647-6168ec80-d31d-11ea-8051-e899a559d859.png">

Reply:
What you're looking at is the header. The split () without anything does white spaces by default. From the front end we send the token like that, but once it comes into the backend it's taken off. U can try it out with a print statement after that.